### ISSUE TITLE
[RFC] Require \Timestampable model behavior only take one field

### ISSUE THREAD

**Issue by SidRoberts** [phalcon/cphalcon#14037]
Something that I found in https://github.com/phalcon/cphalcon/pull/14027 is that the Timestampable model behavior has the option to pass a string or an array of fields:

https://github.com/phalcon/cphalcon/blob/7d8ffd16f6f44bf28d0d700b34cfd6121494023a/phalcon/Mvc/Model/Behavior/Timestampable.zep#L80

If you pass an array, those fields will be updated with the same timestamp and in the same format:

```
+-----+---------------------+---------------------+
| ID  | Timestamp1          | Timestamp2          |
|-----|---------------------|---------------------|
| 123 | 2019-05-04 08:55:23 | 2019-05-04 08:55:23 |
+-----+---------------------+---------------------+
```

Why would anyone need to legitimately use this? Is it safe to remove passing as an array?

**Comment by sergeyklay** [comment]
I could be wrong, but it seems @Jurigag can help us here.

**Comment by JABirchall** [comment]
>  Allows to automatically update a model’s attribute saving the datetime when a record is created or updated

So it functions the same as laravels timestamp attributes
```php
private $timestampable = [
    'created_at',
    'updated_at',
];
```

**Comment by SidRoberts** [comment]
Yeah, I think on creation, only `createdAt` should be altered. Technically, it hasn't been updated so `updatedAt` shouldn't have a value.

**Comment by niden** [comment]
I am fine with this being changed to only one field per model.
